index.html

```diff
@@ -8,10 +8,30 @@
     <title>Portfolio</title>
 </head>
 <body>
-    <nav>
-        <ul>
-            <li>Logo</li>
-        </ul>
-    </nav>
+    <div id="header-background">
+        <nav id="flex-nav">
+            <ul>
+                <li id="logo"><a href="index.html">My Logo</a></li>
+                <li id="burguer-menu-icon ">
+                    <img src="img/menu.svg" alt="burguer-menu" class="burguer-menu-vector">
+                </li>
+            </ul>
+        </nav>
```

Comment:
Please it is advisable to use the `header` tag to hold the logo and the nav menu (hamburger). Header tags are helpful for users and search engines. For your users, they give them a preview of the content they're about to read. For search engines like Google, they provide context on what your page is all about and provide a hierarchy.